### DISCUSSION TITLE
Build custom connector in Release mode with VSCode

### DISCUSSION THREAD

**Discussion by hernerj** [microsoft/vscode-powerquery-sdk#237] [General]
By default, when I create a new custom connector project, the .mez file that gets built is placed in a bin/AnyCPU/Debug folder under the project directory. This implies that the file is being built in a debug configuration. However, I can’t find anywhere to change the configuration to release. I tried poking around in Terminal -> Configure Tasks and looking at the task.json file that is used when the Evaluate Current File button is clicked but I don't see any references to the build configuration anywhere. So, my question is, can you build a connector in release configuration? Or is there really no such thing as debug vs release configuration for the custom connector and the file just happens to go into a folder called debug by default?

**Comment by mattmasson** [comment]
There is no difference between "Release" and "Debug" when it comes to building M extensions. The .mez file is just a zip containing your M code and resources.

We kept the msbuild model of using these configuration directories so that the connector project files fit in with larger projects (that contain more than just your M connectors).

We might simplify that for the "SDK Build" option in the future (for people who are just working on a single connector and have no need to use msbuild features). But we decided to keep the two build modes similar for the time being.

**Reply by hernerj** [reply]
Thanks for the response!
